client/src/components/Ticketing/ticketingmanager/SubscriptionRedemption/SubscriptionTicketItemRow.tsx

```diff
@@ -0,0 +1,289 @@
+                  eventinstances.some(
+                    (instance) =>
+                      instance.availableseats &&
+                      instance.ticketrestrictions.some(
+                        (res) => res.availabletickets,
+                      ) &&
+                      (!eventRestriction || !currentEvents.has(option.eventid)),
+                  ),
+              )
+              .map((option) => ({
+                id: option.eventid,
+                description: option.eventname,
+              }))
+              .concat({id: -1, description: 'Select Event'})}
+          />
+        </td>
+        <td className='p-1'>
+          <Field
+            name={`${field.name}.eventinstanceid`}
+            component={OptionSelect}
+            disabled={field.value.eventid === -1}
+            handleChange={getHandleEventInstanceChange(
+              options,
+              setOptions,
+              field.name,
+              setFieldValue,
+              field.value,
+            )}
+            options={[{id: -1, description: 'Select Showing'}].concat(
+              event?.eventinstances
+                .filter(
+                  (instance) =>
+                    instance.eventinstanceid === field.value.eventinstanceid ||
+                    (instance.availableseats > 0 &&
+                      instance.ticketrestrictions.some(
+                        (res) => res.availabletickets,
+                      )),
+                )
+                .map((instance) => ({
+                  id: instance.eventinstanceid,
+                  description: `${format(
+                    getDate(instance.eventdate, instance.eventtime),
+                    'MM/dd/yyyy hh:mm aa',
+                  )} ${instance.ispreview? '(preview)': ''}`,
+                })) ?? [],
+            )}
+          />
+        </td>
+        <td className='p-1'>
+          <Field
+            name={`${field.name}.ticketrestrictionid`}
+            component={OptionSelect}
+            disabled={field.value.eventinstanceid === -1}
+            handleChange={getHandleTicketRestrictionChange(
+              options,
+              setOptions,
+              field.name,
+              setFieldValue,
+              field.value,
+            )}
+            options={[{id: -1, description: 'Select Ticket Type'}].concat(
+              eventInstance?.ticketrestrictions
+                .filter(
+                  (res) =>
+                    res.availabletickets > 0 ||
+                    res.ticketrestrictionsid ===
+                      field.value.ticketrestrictionid,
+                )
+                .map((res) => ({
+                  id: res.ticketrestrictionsid,
+                  description: res.description,
+                })) ?? [],
+            )}
+          />
+        </td>
+        <td>
+          <FormButton
+            testID='remove-subscription-row-button'
+            title='remove row'
```

Comment:
Do you think you could make this `Remove Ticket`, and usually for our labels, we won't start with a lowercase word.

Reply:
Since it is such a small button do you think it would make sense to have it just say Remove?

Reply:
Yeah, that makes sense

Reply:
Updated